<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Creator Administrator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../figures/logo.ico</normaloff>../../../../../figures/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="JobsQTabWidget" name="jobsQTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="allJobsTab">
       <attribute name="title">
        <string>All Jobs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QStackedWidget" name="allQStackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="jobsListPage">
             <property name="job_name" stdset="0">
              <string>Noneeee</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="LaserAllJobsOverviewQListWidget" name="allJobsQListWidget">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wachtrijTab">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wachtrij&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Wachtrij</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QStackedWidget" name="wachtrijQStackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="jobsListPage_2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="LaserWachtrijJobsOverviewQListWidget" name="wachtrijJobsQListWidget"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="jobPage_2">
             <property name="job_name" stdset="0">
              <string>Noneeee</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="jobQLabel">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;job_name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="LaserJobContentQListWidget" name="listWidget_10">
                  <property name="toolTip">
                   <string>Shortcut: Esc</string>
                  </property>
                  <property name="toolTipDuration">
                   <number>4</number>
                  </property>
                  <property name="statusTip">
                   <string>statustip</string>
                  </property>
                  <property name="whatsThis">
                   <string>what is ths</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="BackQPushButton" name="backPushButton_2">
                    <property name="toolTip">
                     <string extracomment="Shortcut: Esc">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: Esc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="text">
                     <string>Back</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="LaserKlaarQPushButton" name="pushButton_3">
                    <property name="toolTip">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: Ctrl+k&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="text">
                     <string>Laser Klaar</string>
                    </property>
                    <property name="shortcut">
                     <string>Ctrl+K</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="LaserAfgekeurdQPushButton" name="pushButton_4">
                    <property name="text">
                     <string>Afgekeurd</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="LaserOptionsQPushButton" name="wachtrijOptionsQPushButton">
                    <property name="text">
                     <string>Options</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wachtrijMateriaalTab">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wachtrij Material&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Wachtrij Material</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QStackedWidget" name="wachtrijMateriaalQStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="jobsListPage_3">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="LaserMaterialOverviewQListWidget" name="wachtrijMateriaalJobsQListWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="jobPage_3">
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="materialQLabel">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;material_name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="LaserMaterialContentQListWidget" name="listWidget_7"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="BackQPushButton" name="backPushButton_3">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: Esc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Back</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="MateriaalKlaarQPushButton" name="pushButton_5">
                <property name="text">
                 <string>Materiaal Klaar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="LaserOptionsQPushButton" name="wachtrijMateriaalOptionsQPushButton">
                <property name="text">
                 <string>Options</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verwerktTab">
       <attribute name="title">
        <string>Verwerkt</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QStackedWidget" name="verwerktQStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="jobsListPage_4">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="LaserVerwerktJobsOverviewQListWidget" name="verwerktJobsQListWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="jobPage_4">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="jobQLabel_2">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;job_name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="LaserJobContentQListWidget" name="listWidget_8"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="BackQPushButton" name="backPushButton_4">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: Esc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="text">
                   <string>Back</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="LaserOptionsQPushButton" name="verwerktOptionsQPushButton">
                  <property name="text">
                   <string>Options</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="afgekeurdTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Afgekeurd</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QStackedWidget" name="afgekeurdQStackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="jobsListPage_5">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="LaserAfgekeurdJobsOverviewQListWidget" name="afgekeurdJobsQListWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="jobPage_5">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="jobQLabel_3">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;job_name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="LaserJobContentQListWidget" name="listWidget_9"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="BackQPushButton" name="backPushButton_5">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: Esc&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="text">
                   <string>Back</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="LaserOptionsQPushButton" name="afgekeurdOptionsQPushButton">
                  <property name="text">
                   <string>Options</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="createJobsMenu">
    <property name="title">
     <string>Create Jobs</string>
    </property>
    <addaction name="importFromMailAction"/>
    <addaction name="selectFilesAction"/>
    <addaction name="selectFoldersAction"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="editSettingsAction"/>
    <addaction name="checkHealthAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="openDocumentationAction"/>
    <addaction name="reportaBugAction"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="createJobsMenu"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="importFromMailAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import from Mail</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="selectFilesAction">
   <property name="text">
    <string>Select Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="showDocumentationAction">
   <property name="text">
    <string>Show Docs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPMMA_Laser_cutter">
   <property name="text">
    <string>PMMA Laser cutter</string>
   </property>
  </action>
  <action name="actionMetal_Laser_Cutter">
   <property name="text">
    <string>Metal Laser Cutter</string>
   </property>
  </action>
  <action name="selectFoldersAction">
   <property name="text">
    <string>Select Folders</string>
   </property>
  </action>
  <action name="editSettingsAction">
   <property name="text">
    <string>Edit Settings</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open dialog to change the settings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="openDocumentationAction">
   <property name="text">
    <string>Open Documentation</string>
   </property>
  </action>
  <action name="show3DPrintDocumentationAction">
   <property name="text">
    <string>3D printer Documentation</string>
   </property>
  </action>
  <action name="reportaBugAction">
   <property name="text">
    <string>Report a Bug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Creator Administrator</string>
   </property>
  </action>
  <action name="checkHealthAction">
   <property name="text">
    <string>Check Health</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Compares jobs on file system with jobs in tracker file.&lt;/p&gt;&lt;p&gt;Synchronizes/Repair if they  if they are out of sync.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BackQPushButton</class>
   <extends>QPushButton</extends>
   <header>button.h</header>
  </customwidget>
  <customwidget>
   <class>JobsQTabWidget</class>
   <extends>QTabWidget</extends>
   <header>tab_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LaserKlaarQPushButton</class>
   <extends>QPushButton</extends>
   <header>laser_button.h</header>
  </customwidget>
  <customwidget>
   <class>MateriaalKlaarQPushButton</class>
   <extends>QPushButton</extends>
   <header>laser_button.h</header>
  </customwidget>
  <customwidget>
   <class>LaserAfgekeurdQPushButton</class>
   <extends>QPushButton</extends>
   <header>laser_button.h</header>
  </customwidget>
  <customwidget>
   <class>LaserMaterialOverviewQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserMaterialContentQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserWachtrijJobsOverviewQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserJobContentQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserAllJobsOverviewQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserVerwerktJobsOverviewQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserAfgekeurdJobsOverviewQListWidget</class>
   <extends>QListWidget</extends>
   <header>laser_qlist_widget.h</header>
  </customwidget>
  <customwidget>
   <class>LaserOptionsQPushButton</class>
   <extends>QPushButton</extends>
   <header>laser_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
